examples/with-compiled-css/.babelrc

```diff
@@ -0,0 +1,4 @@
+{
+  "presets": ["@babel/preset-react"],
```

Comment:
I get `Error: Cannot find module '@babel/preset-react'`

Based on the official [Next.js Customizing Babel Config](https://nextjs.org/docs/advanced-features/customizing-babel-config) this should be:

```
{
  "presets": ["next/babel"],
  ...
}
```